components.css

```diff
@@ -27,9 +27,9 @@
     --background-color        :#F3F3F3;
     --extra-orange-color      :#EF836D;
     --compilmentary-blue-color:#1AB3B3;
```

Comment:
Typo - fix "compilmentary" to "complimentary"

Reply:
Done, thanks for review.